packages/web3torrent/src/library/paid-streaming-extension.ts

```diff
@@ -61,8 +74,13 @@ export abstract class PaidStreamingExtension implements Extension {
       this.peerAccount = handshake.pseAccount.toString();
     }
 
+    if (handshake.outcomeAddress) {
+      this.peerOutcomeAddress = handshake.outcomeAddress.toString();
```

Comment:
Why is `toString` necessary?

Reply:
Because the fields in the handshake are not simple strings.
AFAIK, the fields/props are bencoded, or encoded in some way.
Without the .toString() that handshake.outcomeAddress looks like this:
```
Uint8Array(42) [48, 120, 100, 54, 101, 50, 54, 100, 56, 97, 99, 102, 100, 50, 57, 52, 56, 99, 48, 54, 48, 57, 56, 99, 54, 100, 101, 51, 56, 54, 99, 56, 57, 98, 49, 50, 101, 48, 102, 57, 49, 54]
```